iphone/Classes/TiRootViewController.m

```diff
@@ -209,7 +211,19 @@ - (void)remoteControlReceivedWithEvent:(UIEvent *)event
 -(void)updateBackground
 {
 	UIView * ourView = [self view];
-	UIColor * chosenColor = (bgColor==nil)?[UIColor blackColor]:bgColor;
+	UIColor * chosenColor = bgColor;
+
+	if (chosenColor == nil) {
+#if defined(DEFAULT_BGCOLOR_RED) && defined(DEFAULT_BGCOLOR_GREEN) && defined(DEFAULT_BGCOLOR_BLUE)
+		chosenColor = [UIColor colorWithRed: DEFAULT_BGCOLOR_RED
+									  green: DEFAULT_BGCOLOR_GREEN
+									   blue: DEFAULT_BGCOLOR_BLUE
+									  alpha: 1.0f];
+#else
+		chosenColor = [UIColor blackColor];
+#endif
+	}
```

Comment:
Not sure what you mean. `chosenColor` is either `bgColor` or black. If `bgColor` is `nil`, then `chosenColor` is `nil`. If `chosenColor` is `nil`, then we used to set it to black, but now we check if the colors are defined otherwise fallback to black. The code is good. Not sure what your concern is.